be/src/storage/lake/versioned_tablet.cpp

```diff
@@ -102,4 +104,50 @@ std::vector<RowsetPtr> VersionedTablet::get_rowsets() const {
     return Rowset::get_rowsets(_tablet_mgr, _metadata);
 }
 
+void VersionedTablet::get_basic_info(TabletBasicInfo& info) const {
```

Comment:
it's a little strange that the return type is void, using reference parameter to pass info out.

`TabletBasicInfo VersionedTablet::get_basic_info() const` shall be more straight forward.